pkg/operator/vitesstopo/prune_cells.go

```diff
@@ -80,10 +80,11 @@ func DeleteCells(ctx context.Context, ts *topo.Server, recorder record.EventReco
 	resultBuilder := &results.Builder{}
 
 	for _, cellName := range cellNames {
-		if err := ts.DeleteCellInfo(ctx, cellName); err != nil {
+		// topo.NoNode is the error type returned if we can't find the cell when deleting. This ensures that this operation is idempotent.
```

Comment:
Do you think that it would be safer to handle the `NoNode` error distinctly? I feel that it would be cleaner and would read better if this function was modified in the future.

Reply:
My feeling is that the first conditional arm is our error path - and fundamentally a "not found" error should be a non-error in a delete operation, because delete operations should be idempotent. In this case, I think it makes more sense to combine them together to very clearly illustrate that an error is only an error if it's NOT a "not found" error specifically in relation to delete operations.